lib/l10n/intl_en.arb

```diff
@@ -97,6 +98,7 @@
   "actionOpenFilter": "Open filter",
   "actionRequestPermissions": "Request permissions",
   "actionRefresh": "Refresh",
+  "actionSource": "Information sources",
```

Comment:
This isn't really an action, it may fit better under sections ("sectionInformationSources").